data/planet.lua

```diff
@@ -451,7 +516,7 @@ local space_connections = {
         from = "mercury",
         to = "venus",
         order = "a[mercury]-b[venus]",
-        length = 5.0, -- 水星到金星的距离，比例单位
+        length = 869585.66, -- Average distance between mercury and venus in units of 100 kilometers.
```

Comment:
Make lengths accurate to real life. This scheme is replicated down to every planet between here and the Inner Asteroid Belt to Jupiter space connection.